drivers/misc/Kconfig

```
@@ -553,6 +553,11 @@ config I2C_EEPROM
 	help
 	  Enable a generic driver for EEPROMs attached via I2C.
 
+config RADXA_ID_EEPROM
+       bool "Radxa eeprom Support"
```

```suggestion
       bool "Enable Radxa ID EEPROM driver"
```
另外这个Radxa ID EEPROM名字感觉很怪，platform data之类的可能更适当一些